src/runtime/internal/ssr.ts

```diff
@@ -32,16 +32,31 @@ export function spread(args, classes_to_add) {
 	return str;
 }
 
-export const escaped = {
+const ATTR_REGEX = /[&"]/g;
+const CONTENT_REGEX = /[&<]/g;
+
+const escapes = {
 	'"': '&quot;',
-	"'": '&#39;',
 	'&': '&amp;',
-	'<': '&lt;',
-	'>': '&gt;'
+	'<': '&lt'
 };
 
-export function escape(html) {
-	return String(html).replace(/["'&<>]/g, match => escaped[match]);
+export function escape(html: string, attr: 0 | 1 = 0) {
```

Comment:
any reason not to make `attr` a `boolean`? at first I thought it was a size saving technique, but this is only used in ssr, so that probably wouldn't be necessary

Reply:
space-saving, yeah — just force of habit really. though is there a good reason _to_ use a boolean here? bytes are less expensive on the server but they're still not free.

Reply:
I was just writing a comment that I then realized was the same as Ben's comment. Since this is SSR-only, this space saving technique seems unnecessary.

Reply:
Oh, your comment hadn't appeared yet when I commented for some reason, Rich. I guess I would say that if someone were serious about their bytes on the server, they'd minify, and in that case `true` would be replaced by `!0` which is almost as short as `1`. I'd say the good reason to use a boolean is that it's easier to understand.

Either way, I'd probably be in favor of calling this argument something like `is_attr` instead.

Reply:
I like that, will update the PR in the morning

Reply:
done. this should be good to merge now